docs/src/apalache/features.md

```diff
@@ -2,24 +2,24 @@ Here is the list of the TLA+ language features that are currently supported by A
 
 ## Safety vs. Liveness
 
-At the moment, Apalache is able to check invariants and inductive invariants.
-It means that you can only check safety with Apalache, unless you employ
-[liveness-to-safety] transformation in your spec. It does not support
-liveness properties.  If you really like to see liveness implemented, upvote
-the [liveness feature].
+At the moment, Apalache is only able to check invariants and inductive
+invariants.  Which means that you can only check safety properties with
+Apalache, unless you employ a [liveness-to-safety] transformation in your spec.
+In general, we doe not support checking liveness properties.  If you would like
+to see liveness implemented, upvote the [liveness feature].
 
 ## Language
 
 ### Module-Level constructs
 
 Construct  | Supported? | Milestone | Comment
 ------------------------|:------------------:|:---------------:|--------------
-``EXTENDS module`` | ✔ | - | As soon as SANY imports the module. Some standard modules are not supported yet
-``CONSTANTS C1, C2`` | ✔ | -  | Either define a ``ConstInit`` operator to initialize the constants, or declare operators instead of constants, e.g., C1 == 111
+``EXTENDS module`` | ✔ | - | As soon as SANY imports the module. <!-- What does the prev sentence mean? --> Some standard modules are not supported yet
```

Comment:
I don't understand what is meant in this context by the sentence 

> As soon as SANY imports the module.

Reply:
I also don't understand that. It is all modulo SANY parsing things :)

Reply:
Let's remove that sentence